course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -130,7 +136,54 @@ describe('Unit testing PeopleController', function() {
             $httpBackend.expectGET(courseInstanceURL)
                         .respond(200, JSON.stringify(ci));
             $httpBackend.flush(1);
-            expect(scope.title).toEqual(ci.title);
+            expect(scope.courseInstance['title']).toEqual(ci.title);
+            expect(scope.courseInstance['school']).toEqual(ci.course.school_id);
+        });
+    });
+
+    describe('getFormattedCourseInstance', function() {
+        var ci;
+        beforeEach(function() {
+
+            ci = {
+                course_instance_id: '1234',
+                title: 'Test Title',
+                sites: [
+                    {
+                        external_id: 'https://x.y.z/888',
+                        site_id: '888',
+                    }
+                ],
+                course :{
+                    school_id: 'ABC',
```

Comment:
I'd make this lower-case, so we're sure it's uppercasing the raw value correctly.